frontend/src/component/common/FormTemplate/FormTemplate.tsx

```diff
@@ -34,6 +34,7 @@ const StyledContainer = styled('section', {
     minHeight: modal ? '100vh' : '80vh',
     borderRadius: modal ? 0 : theme.spacing(2),
     width: '100%',
+    height: '100%',
```

Comment:
Not sure why this was not set before, but it seems to provide a more straightforward behavior without having to set the modal prop.